panda/src/cocoadisplay/cocoaGraphicsWindow.mm

```diff
@@ -626,13 +626,40 @@
     }
   }
 
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 1060
```

Comment:
Does this change mean we completely ignore fullscreen on versions prior to 10.6?

Reply:
Oh, good catch! That check is at the wrong place and so breaks the 10.6 build :(